### PR TITLE
v3

### LINKED ISSUE #218: Update Barge for v3
Have a lightweight barge for v3, with only this components:
  - ganache
 - ocean-contracts
 - aquarius + elasticsearch
 - provider-py

Also, you should be able to start_ocean with a custom RPC, this reducing the need to run ganache if alternate network is provided.

### PR DESCRIPTION
**Pull request by kremalicious**
It changes everything™. Just not the dolphin. The [README changeset](https://github.com/oceanprotocol/barge/pull/219/files#diff-04c6e90faac2675aa89e2176d2eec7d8) is the best place to see what got changed. The `market` UI is not included in the beginning, so `barge` is mainly used for local development of `market`.

This branch is currently in use for integration testing of:

- `ocean.js`
- `ocean.py`
- `react`
- `market`

Screen Shot 2020-09-08 at 18 34 35

Closes #218

### CODE REVIEW

File: compose-files/provider.yml [hunk 1]
```diff
@@ -24,5 +24,6 @@ services:
       IPFS_GATEWAY: ${PROVIDER_IPFS_GATEWAY}
       OCEAN_PROVIDER_TIMEOUT: '9000'
       OPERATOR_SERVICE_URL: ${OPERATOR_SERVICE_URL}
+      AQUARIUS_URL: ${AQUARIUS_URL:-http://172.15.0.5:5000}
```
Comment (kremalicious) [inline compose-files/provider.yml:27]: we have `AQUARIUS_URL` & `AQUARIUS_URI`? https://github.com/oceanprotocol/barge/blob/v3/start_ocean.sh#L84

We should only have one, so just use `AQUARIUS_URI` everywhere, no need to set another default here

File: compose-files/aquarius.yml [hunk 2]
```diff
@@ -18,3 +20,15 @@ services:
       DB_CA_CERTS: ${DB_CA_CERTS}
       DB_CLIENT_KEY: ${DB_CLIENT_KEY}
       DB_CLIENT_CERT: ${DB_CLIENT_CERT}
+      LOG_LEVEL: DEBUG
+      EVENTS_ECIES_PRIVATE_KEY: "0x5d75837394b078ce97bc289fa8d75e21000573520bfa7784a9d28ccaae602bf8"
+      ARTIFACTS_PATH: "/ocean-contracts/artifacts"
+      ADDRESS_FILE: ${OCEAN_ADDRESS_FILE:-/ocean-contracts/artifacts/address.json}
```
Comment (kremalicious) [inline compose-files/aquarius.yml:26]: `OCEAN_ADDRESS_FILE` only exists here. Is this supposed to be `ADDRESS_FILE`, the one from the main script?

File: start_ocean.sh [hunk 3]
```diff
@@ -82,61 +73,22 @@ export DB_CLIENT_KEY=""
...
-# private key for agent, public address: 0x6f2b82bB771687b69d0932c7386742804144ae7D
-# NEVER USE this address in production !
-export AGENT_PRIVATE_KEY='axis talent grab cushion figure couple plug ostrich file false jealous nest'
-
-# Export User UID and GID
-export LOCAL_USER_ID=$(id -u)
-export LOCAL_GROUP_ID=$(id -g)
-
 
 #add aquarius to /etc/hosts
```
Comment (kremalicious) [inline start_ocean.sh:93]: do we still have to modify every user's `/etc/host` file?

**Review by alexcos20 — APPROVED** [review]
lgtm

**Pull request merged**